client/app/views/menu_item.coffee

```diff
@@ -97,6 +102,15 @@ module.exports = class MenuItemView extends BaseView
             trashButton.removeClass 'hidden'
 
 
+    onCalendarChange: =>
```

Comment:
You can safely use single arrow `->`, the `listenTo` statement (L27) keep the func binding